remove_dead_rate: 0.2